scummvm/scummvm
# VKEYBD: Repair the apostrophe key in the virtual keyboard

The `target` and `name` attributes for the apostrophe key (straight `'` in ASCII, not the real typographic apostrophe) didn't use the same character in the XML tables, so they never matched and this particular key had no effect in the virtual keyboard.

To test this PR:

1. Build ScummVM with `--enable-vkeybd`
2. Make sure that you have a valid `vkeybdpath` value pointing to the new files (such as `/path/to/git/scummvm/backends/vkeybd/packs`) in your configuration file
3. Open the virtual keyboard within ScummVM (default shortcut on desktop is Ctrl-F7)
4. Click on the apostrophe key on the virtual keyboard (immediately on the left of the Enter key)
5. Make sure that it outputs a proper `'` character (it used to do nothing before this)

(The zip files were rebuilt with `vkeybdpack.py makeall`; the biggest one went 6KB larger, so I added `compresslevel=9` to the Python script in order to go back to the previous sizes)
